AntennaTracker/Tracker.h

```diff
@@ -172,7 +172,7 @@ class Tracker : public AP_HAL::HAL::Callbacks {
         uint32_t last_update_ms;    // last position update in milliseconds
         Vector3f vel;           // the vehicle's velocity in m/s
         int32_t relative_alt;	// the vehicle's relative altitude in meters * 100
-    } vehicle;
+    } vehicle {};
```

Comment:
You can remove this change. It's already guaranteed to be zero'd, and the initializer is actually more code sometimes.

Reply:
ok, nice !